src/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -51,6 +51,11 @@ public bool IsSingleSegment
         /// </summary>
         public ReadOnlyMemory<T> First => GetFirstBuffer();
 
+        /// <summary>
+        /// Gets <see cref="ReadOnlySpan{T}"/> from the first segment.
+        /// </summary>
+        public ReadOnlySpan<T> FirstSpan => First.Span;
```

Comment:
I thought the point of adding this property was that it would be more optimized than just doing `First.Span`.  If it's just `First.Span`, it's saving the caller having to type a single '.', which isn't worth a new property.  Did I misunderstand?

Reply:
You are correct. I was confused also about what the value add for using First.Span is. I opened the PR to make sure what the exact ask is. 
I had the conversation with Ahson and will make the proper improvements on this in a bit.

Reply:
Ok. Thanks.